include.js

```diff
@@ -2299,6 +2299,12 @@ let cards = [
     githubLink: 'https://github.com/vishalpatil20'
   },
   {
+    artName: 'Moving and rotating galaxy ',
+    pageLink: './Art/supriya/galaxy.html',
+    imageLink: './Art/supriya/galaxy.gif',
```

Comment:
Please move your artwork into a folder that has the same name as your github username and adjust the details here.